src/constants.ts

```diff
@@ -11,15 +11,15 @@ export const copyNumberCat = [
   {value: 0, name: 'NORMAL', color: '#dcdcdc', border: 'transparent'},
   //{value: 1, name: 'Low level amplification', color: '#f4a582'},
   //{value: 2, name: 'High level amplification', color: '#ca0020'},
-  {value: 'null', name: 'Unknown', color: '#FCFCFC', border: '#dcdcdc'}
+  // {value: 'null', name: 'Unknown', color: '#FCFCFC', border: '#dcdcdc'}
 ];
-export const unknownCopyNumberValue: any = copyNumberCat[copyNumberCat.length-1].value;
+export const unknownCopyNumberValue: any = 'null';
 
 export const mutationCat = [
   {value: 'true', name: 'Mutated', color: '#1BA64E', border: 'transparent'},
   {value: 'false', name: 'Non Mutated', color: '#aaa', border: 'transparent'},
-  {value: 'null', name: 'Unknown', color: 'transparent', border: '#999'}
+  // {value: 'null', name: 'Unknown', color: 'transparent', border: '#999'}
```

Comment:
We cannot remove this category as we are using them also in the OncoPrints.

https://github.com/Caleydo/tdp_gene/blob/2ff1964b81f4651fc7367c6564b13b7ae967e644/src/views/AOncoPrint.ts#L245-L247